jerry-core/api/jerry.c

```diff
@@ -6426,6 +6427,135 @@ jerry_get_container_type (const jerry_value_t value) /**< the container object *
   return JERRY_CONTAINER_TYPE_INVALID;
 } /* jerry_get_container_type */
 
+/**
+ * Return a new array containing elements from a Container or a Container Iterator.
+ * Sets the boolean input value to `true` if the container object is a
+ * key-value structure and `false` if not.
+ *
+ * Note:
+ *     the returned value must be freed with a jerry_release_value call
+ *
+ * @return jerry_value_t representing an array containing the elemets of the map.
+ *         jerry_value_t representing an empty array, if the Iterator is finished/empty.
+ *         Error if the value is not a Container or Container Iterator.
+ *         `undefined` if the `value` is undefined or null.
```

Comment:
I would siplify this:
`@ return an array of items for maps/sets or their iterators, error otherwise`
Passing an undefined as value is an error.